crates/oxc_linter/src/snapshots/no_find_dom_node.snap

```diff
@@ -0,0 +1,38 @@
+---
+source: crates/oxc_linter/src/tester.rs
+assertion_line: 105
+expression: no_find_dom_node
+---
+  ⚠ Do not use findDOMNode. It doesn’t work with function components and is deprecated in StrictMode. See https://reactjs.org/docs/react-dom.html#finddomnode
+   ╭─[no_find_dom_node.tsx:3:1]
+ 3 │               componentDidMount: function() {
+ 4 │                 React.findDOMNode(this).scrollIntoView();
+   ·                 ───────────────────────
```

Comment:
Can you try and point the span to just `findDOMNode`, it's the smallest span on where the error is.

Reply:
Sure, it is done in a7bf448f057290468af089f1d779bae249b899a8.